### PR TITLE
adding SonarCloud steps

### PR DESCRIPTION
**Pull request by elduddz**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -9,18 +9,27 @@ stages:
...
-              AWS_DEFAULT_REGION: $(AWS_DEFAULT_REGION)
-              AWS_SECRET_ACCESS_KEY: $(AWS_SECRET_ACCESS_KEY)
-              TF_VAR_PASSWORD: $(TF_VAR_PASSWORD)
-
+        - script: ./gradlew clean check
+          env:
+            AWS_ACCESS_KEY_ID: $(AWS_ACCESS_KEY_ID)
+            AWS_DEFAULT_REGION: $(AWS_DEFAULT_REGION)
+            AWS_SECRET_ACCESS_KEY: $(AWS_SECRET_ACCESS_KEY)
+            TF_VAR_PASSWORD: $(TF_VAR_PASSWORD)
```
Comment (ciaranevans) [inline azure-pipelines.yml]: I think those env vars can go. Running it locally without them seems fine.

Comment (JoelClemence) [reply azure-pipelines.yml]: BUT Locally we probably have a ~/.aws/credentials file which is where the aws cli/sdk get the credentials from. More info: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html

Comment (elduddz) [reply azure-pipelines.yml]: ah, those vars arent mine, they was there when I got there :)

Comment (ciaranevans) [reply azure-pipelines.yml]: @JoelClemence indeed we do, but the check doesn't need those :)

**Pull request merged**